scripts/build-brotli.sh

```diff
@@ -103,14 +103,22 @@ if $BUILD_WASM; then
     make -j
     make install
     cp -rv "$TEMP_INSTALL_DIR_ABS/lib" "$TARGET_DIR_ABS/lib-wasm"
+    (cd "$TARGET_DIR_ABS/lib-wasm" && \
+        [ -f libbrotlicommon.a ] && ln -sf libbrotlicommon.a libbrotlicommon-static.a || true; \
+        [ -f libbrotlidec.a ] && ln -sf libbrotlidec.a libbrotlidec-static.a || true; \
+        [ -f libbrotlienc.a ] && ln -sf libbrotlienc.a libbrotlienc-static.a || true)
     cd ..
 fi
 
 if $BUILD_LOCAL; then
     mkdir -p buildfiles/build-local
     cd buildfiles/build-local
-    cmake ../../ -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="$TARGET_DIR_ABS"
+    cmake ../../ -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="$TARGET_DIR_ABS" -DBUILD_SHARED_LIBS=OFF
     make -j
     make install
+    (cd "$TARGET_DIR_ABS/lib" && \
```

Comment:
is this needed? Maybe just our submodule commit was wrong?

Reply:
I removed it to check if it goes through with the update brotli submodule commit

Reply:
[67822bd](https://github.com/EspressoSystems/nitro-espresso-integration/pull/561/commits/67822bd4dec51d16f56c099a85c65c364eea576b)